src/interpreters/mitie_sklearn_interpreter.py

```diff
@@ -18,12 +23,23 @@ def get_entities(self, tokens):
         return d
 
     def get_intent(self, tokens):
-        label, _ = self.classifier(tokens)  # don't use the score
-        return label
+        """Returns the most likely intent and its probability for the input text.
+
+        :param text: text to classify
+        :return: tuple of most likely intent name and its probability"""
+        if self.classifier:
+            X = self.featurizer.create_bow_vecs(tokens)
```

Comment:
please add a comment why tokens are passed to this function although the function expects sentences.